src/vscode-dts/vscode.d.ts

```diff
@@ -15638,9 +15638,13 @@ declare module 'vscode' {
 
 	/**
 	 * Namespace for localization-related functionality in the extension API. To use this properly,
-	 * you must have `l10n` defined in your `package.json` and have bundle.l10n.<language>.json files.
+	 * you must have `l10n` defined in your extension manifest and have bundle.l10n.<language>.json files.
 	 * For more information on how to generate bundle.l10n.<language>.json files, check out the
 	 * [vscode-l10n repo](https://github.com/microsoft/vscode-l10n).
+	 *
+	 * Note: Built-in extensions (for example, Git, TypeScript Language Features, GitHub Authentication)
```

Comment:
I'm wondering whether this should go in internal documentation somewhere- the intended reader of the vscode.d.ts might more be external extension authors.